frontend/src/components/dashboard/components/categories-list/components/category/category.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from 'common/types/types';
+import { Image } from 'components/common/common';
+
+import { getRandomColor } from './helpers/helpers';
+import styles from './styles.module.scss';
+
+type Props = {
+  keyName: string;
+  name: string;
+};
+
+const Category: FC<Props> = ({ keyName, name }) => {
+  return (
+    <div className={styles.category} style={{ borderColor: getRandomColor() }}>
+      <Image
+        width={'30px'}
+        height={'30px'}
```

Comment:
```suggestion
        width="30px"
        height="30px"
```
don't need curly for strings